### PR TITLE
Button

### PR DESCRIPTION
**Pull request by rahulcs754**
https://user-images.githubusercontent.com/6254980/151715330-8dd5fae9-4938-4068-a647-8ed5b69b3d9a.mp4

Please  check and give your review

### CODE REVIEW

File: css/components/button.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.btn-primary{
+    background-color: var(--primary-color);
+    color: var(--mint-white);
+}
+
+.btn-primary:hover{
+    background-color: var(--light-blue);
+}
+
+.btn-secondary{
```
Comment (ttahm3d) [inline css/components/button.css]: Dont commit CSS classes without any properties.

File: css/components/button.css [hunk 2]
```diff
@@ -13,13 +36,101 @@
...
+}
+
+.btn-warning{
+    background-color: var(--warning-color);
+    color: var(--mint-white);
+}
+
+.btn-warning:hover{
+    background-color: hsl(40,100%,65%);
+    box-shadow: 1px 1px var(--light-black);
```
Comment (ttahm3d) [inline css/components/button.css]: box-shadow property may be a reduandacy. you have defined it on .btn class and here.

**Review by ttahm3d — COMMENTED** [review]
Good work. Have suggested changes.

File: css/components/button.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+.btn-float{
+    margin: 0.2rem;
+    min-width: 2rem;
+    min-height: 2rem;
+    padding: 0.8rem;
+    border-radius: 50%;
+    text-align: center;
+    flex-direction: row;
+}
```
Comment (tanishq20) [inline css/components/button.css:112]: As per the video, I think icons are not in the center. U can try it with flex to center it.

File: css/components/button.css (hunk 2 above)
Comment (rahulcs754) [reply css/components/button.css]: okay i change

Comment (rahulcs754) [reply css/components/button.css]: I  have done with .btn class thanks for review my code

File: css/components/button.css (hunk 1 above)
Comment (rahulcs754) [reply css/components/button.css]: i removed empty class thanks for giving your time . Thanks tahir

File: css/components/button.css (hunk 3 above)
Comment (rahulcs754) [reply css/components/button.css:112]: i changed accoding you

File: css/components/button.css (hunk 2 above)
Comment (rahulcs754) [reply css/components/button.css]: i change that you told me

**Review by rahulcs754 — COMMENTED** [review]
changes done

**Pull request merged**